crates/sncast/src/main.rs

```diff
@@ -740,3 +735,23 @@ fn update_cast_config(config: &mut CastConfig, cli: &Cli) {
         clone_or_else!(cli.wait_timeout, config.wait_params.get_timeout()),
     );
 }
+
+fn get_cast_config(cli: &Cli) -> Result<CastConfig> {
+    let global_config_path = get_global_config_path().unwrap_or_else(|err| {
+        eprintln!("Error getting global config path: {err}");
+        Utf8PathBuf::new()
+    });
+
+    let global_config =
+        load_global_config::<CastConfig>(&Some(global_config_path.clone()), &cli.profile)
+            .unwrap_or_else(|_| {
+                load_global_config::<CastConfig>(&Some(global_config_path), &None).unwrap()
+            });
+
+    let local_config = load_global_config::<CastConfig>(&None, &cli.profile)?;
```

Comment:
It's loading local config. Please rename this function.

Reply:
Renamed. I've also renamed `GlobalConfig` trait. This is no longer the correct name